commands/deployments/tasks/dep_update_task_step_state_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
```

Comment:
Copyright 2021 :-)

Reply:
Copy/paste again, thx!

```suggestion
// Copyright 2021 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
```